src/server/action_handlers/actionkit-rsvp.js

```diff
@@ -0,0 +1,9 @@
+export const displayName = () => 'ActionKit Event RSVP'
+
+export const available = () => {
+  return true
+}
+
+export const processAction = (questionResponse) => {
```

Comment:
I'm not sure where exactly this will be called from, but makes sense to pass in all available context that might be useful.